src/pages/myGeotab/guides/concepts.tsx

```diff
@@ -142,7 +142,7 @@ const resultsAndErrors: ReactNode = (
 );
 
 const authentication: React.ReactNode = (
-    <div className="paragraph">
+    <div id="authentication" className="paragraph">
```

Comment:
This is unnecessary, id's get attached to these nodes via the Accordion & PageContent components